### ISSUE TITLE
@json function is not working

### ISSUE THREAD

**Issue by Eugene-Melbourne** [haidubogdan/netbeans-php-blade-plugin#32]
@json function is not working

this is my code

```
<script>
    @if( !defined('ADDRESS_PARAMS_SET') )
        var ADDRESS_PARAMS = {};
        ADDRESS_PARAMS.COUNTRIES        = @json( $defaults->countries );
    @endif
</script>
```
this is my error message

![image](https://github.com/haidubogdan/netbeans-php-blade-plugin/assets/37518271/f62d9b6e-d037-4d15-9fd0-02fee4ccfccd)

**Comment by haidubogdan** [comment]
Hi, 
Thank you for your observation. 
The issue should be fixed with the new release [https://github.com/haidubogdan/netbeans-php-blade-plugin/releases/tag/nb180-v.2.0.4](https://github.com/haidubogdan/netbeans-php-blade-plugin/releases/tag/nb180-v.2.0.4)

**Comment by Eugene-Melbourne** [comment]
WOW, it was great. Quick fix. Thank you very much

**Comment by josevv28** [comment]
`@JSON` function is not working

Seems to be working

**Comment by haidubogdan** [comment]
Closing the issue ... the fix is available from v2.0.4
